components/apimgt/org.wso2.carbon.apimgt.common.gateway/src/main/java/org/wso2/carbon/apimgt/common/gateway/util/JWTUtil.java

```diff
@@ -80,6 +80,7 @@ public static String getJWSCompliantAlgorithmCode(String signatureAlgorithm) {
      * @param signatureAlgorithm signature algorithm which needs to include in the header
      * @throws JWTGeneratorException
      */
+
     public static String generateHeader(Certificate publicCert, String signatureAlgorithm) throws
             JWTGeneratorException {
 
```

Comment:
Remove the method content and replace it with `generateHeader(Certificate publicCert, String signatureAlgorithm, false)`

Reply:
+1